.github/ISSUE_TEMPLATE/crash-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Crash report
+about: Report a game crash.
+title: My game crashed
+labels: Crash
+assignees: ''
+
+---
+
+<!-- This is a guideline that shall help you to include all the required information we depend on to investigate and fix game-breaking bugs. Please follow the instructions and replace any placeholders that are written in capital letters. Instructions like this comment will not be visible in your report. -->
+
+## System Information
+<!-- Information about the operating system, engine version, game mod and package source are mandatory for investigating and fixing crashes. -->
+
+- Operating System:  OPERATING SYSTEM
+- OpenRA Version: ENGINE VERSION
+- OpenRA Mod: GAME MOD
+- Source: Official download package OR self-compiled OR third-party package
+
+
+## Exception log
+<!-- Please replace the placeholder below with the content of the exception.log file. The three backsticks before and after the placeholder are used for formatting, so don't remove them. -->
```

Comment:
We could include information on where to find the Logs folder here. (A link to the Wiki should be enough I think, like https://github.com/OpenRA/OpenRA/wiki/FAQ#my-game-just-crashed.)